core/include/vecmem/containers/impl/aligned_multiple_placement.ipp

```diff
@@ -0,0 +1,286 @@
+/*
+ * VecMem project, part of the ACTS project (R&D line)
+ *
+ * (c) 2021 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+#include <tuple>
+#include <type_traits>
+
+#include "vecmem/utils/type_traits.hpp"
```

Comment:
I notice that you don't like to put comments into the include statements. But that's the only way in which we can force clang-format to allow us putting our own includes before the standard library includes.

This comes from one of the ATLAS coding rules. That with the includes we should always go "from the most specific to the least specific headers". Of course this leaves a lot of room for interpretation still. But at the very least it means that our own headers need to be included before the standard library ones. (This is mainly to better test that those headers work well by themselves as well.)

Reply:
Oh, so _that_ is why we always comment the includes! I had no idea. :smile: